### PR TITLE
Update schedule for holidays, extend review rotation.

### PR DESCRIPTION
**Pull request by evankanderson**
/assign @rhuss

### CODE REVIEW

File: mechanics/TOC-REVIEW.schedule [hunk 1]
```diff
@@ -43,4 +43,14 @@
 2021-12-02T08:30:00-08:00 | Eventing WG
 2021-12-09T08:30:00-08:00 | Networking WG
 2021-12-16T08:30:00-08:00 | DUX WG
-2021-12-23T08:30:00-08:00 | Security WG
+2022-12-23T08:30:00-08:00 | Security WG
```
Comment (rhuss) [inline mechanics/TOC-REVIEW.schedule:46]: I think we skip this one. Could you please move all entries below by one week ? Thx!

Comment (rhuss) [reply mechanics/TOC-REVIEW.schedule:46]: also `2022-12-23` does not look right either :-)

Comment (evankanderson) [reply mechanics/TOC-REVIEW.schedule]: D'oh -- you're totally right.

I need to make a script for this.

**Comment by evankanderson** [comment]
Updated.

**Review by rhuss — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**